ash/src/vk/prelude.rs

```diff
@@ -59,3 +59,13 @@ impl From<vk::Extent2D> for vk::Rect2D {
 pub unsafe trait TaggedStructure {
     const STRUCTURE_TYPE: vk::StructureType;
 }
+
+#[cfg(feature = "debug")]
+pub(crate) fn wrap_cstr_slice_until_nul(
+    bytes: &[::std::os::raw::c_char],
+) -> Result<&::std::ffi::CStr, core::ffi::FromBytesUntilNulError> {
```

Comment:
Note that this error type was accidentally forgotten to be reexported from `std` and is only available there since 1.73 (took me a sec to figure out why it was listed in https://blog.rust-lang.org/2023/10/05/Rust-1.73.0.html#stabilized-apis), hence this imports from `core`. Perhaps I should import the rest from `core` as well and start the migration process to `no-std` gracefully.

Reply:
Preferring `core`/`alloc` where applicable in new code seems like a fine start!

Reply:
Done that here now!